### ISSUE TITLE
*RuntimeError: More than one of your folds has a prediction for case PETCT_PETCT_113_TP0.nii.gz**

### ISSUE THREAD

**Issue by Preethi2121** [MIC-DKFZ/nnUNet#1942]
Traceback (most recent call last):
  File "/home/xxxi/xxx/envs/nnUNet/bin/nnUNetv2_find_best_configuration", line 8, in <module>
    sys.exit(find_best_configuration_entry_point()) File "/xxx/data/xxxxxx/Code/nnUNet/nnunetv2/evaluation/find_best_configuration.py", line 295, in find_best_configuration_entry_point
    find_best_configuration(dataset_name, model_dict, allow_ensembling=not args.disable_ensembling,
  File "/mnt/data/Preethi/Code/nnUNet/nnunetv2/evaluation/find_best_configuration.py", line 100, in find_best_configuration
    accumulate_cv_results(output_folder, merged_output_folder, folds, num_processes, overwrite)
  File "/xxx/data/xxx/Code/nnUNet/nnunetv2/evaluation/accumulate_cv_results.py", line 40, in accumulate_cv_results
    raise RuntimeError(f'More than one of your folds has a prediction for case {pf}')
**RuntimeError: More than one of your folds has a prediction for case PETCT_PETCT_113_TP0.nii.gz**, The json split file looks correct. No duplication of files between the split, but I have this error when I tried to find the best configuration.  Some of the image files are saved in more than one fold.

_Originally posted by @Preethi2121 in https://github.com/MIC-DKFZ/nnUNet/issues/1561#issuecomment-1933669150_

**Comment by dojoh** [comment]
Hey, I'm not sure if I fully understand. The individual test sets of the cv folds should be disjoint. How come you have the imagefiles in multiple folds?

**Comment by Preethi2121** [comment]
The original split jason file doesn't have duplicates, it was only after
the training, the val folders has duplicated image files

Preethi

**Comment by Preethi2121** [comment]
Hi,

Please find the requested files attached. The commands used were

nnUNetv2_plan_and_preprocess -d 852 --verify_dataset_integrity

nnUNetv2_train 852 3d_cascade_fullres 0 [--npz] (Repeated this for 1,2,3,4
folds)

nnUNetv2_find_best_configuration 852 -c 3d_fullres- with this command I had
the error

Preethi

**Comment by Preethi2121** [comment]
Hi, 

I am here again with another issue. I seen this specific pattern of artifact on the predicted images from the validation set. Do you have any suggestions or experience why this appears. I have this pattern in all my images  of the validation set. 

Please share your sugesstions to avoid this. 

<img width="148" alt="Screenshot 2024-03-01 103010" src="https://github.com/MIC-DKFZ/nnUNet/assets/117740827/90dc596b-ef4a-4af4-aafa-aea2c27c07a4">
<img width="200" alt="Screenshot 2024-03-01 103624" src="https://github.com/MIC-DKFZ/nnUNet/assets/117740827/e7d8b131-bea1-4d5c-b2a8-46e63dfc10a6">

**Comment by Preethi2121** [comment]
Hi,

I am attaching the requested files again.
The dots that you see are from the cross validation images.
Could you please be more specific about what you mean by different
datasets, because I have these patterns in all cross validation images and
they are all from a single dataset.

Preethi

**Comment by hep-raidium** [comment]
Hey, 
I had the same issue ("More than one of your folds has a prediction for case"). Using --no_overwrite when launching the command nnUNetv2_find_best_configuration resolved the problem. 

Best,
Hélène

**Comment by mrokuss** [comment]
Hey @Preethi2121 

Could you resolve your issue in the meantime?

Cheers,
Max

**Comment by Preethi2121** [comment]
yes.
